Java_fibonacci.java

```diff
@@ -1,11 +1,9 @@
-package milestone6;
-
 import java.util.Scanner;
+import java.math.BigInteger;
 
-public class JavaFibonacci {
+public class Java_fibonacci {
```

Comment:
Codacy found an issue: [The class name 'Java_fibonacci' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6357937)

Reply:
In order to keep the file compilable, this can't be solved while keeping the file name pattern as Java classes must have the same name as the file. Should this be changed anyway?

Reply:
These are the automated code reviews which helps to improve the code quality and readability. Try changing according to the codacy bot otherwise, do which ever you finds the best.

Reply:
Honestly I would ignore both warnings because:

- The class name and the file name must be the same or it won't compile, and in this case, I believe it is more important to keep the file name pattern of the repo
- The package declaration in order to work the file must be inside a folder with the same name, and the package declaration does not accepts the char "-" which means we would have to put the java file inside a folder, which again would go against the pattern of the repo. 

Do you agree?

Reply:
I totally agree with you! Thank you for your contribution. Before I merge your PR. Please add your name into Contributors.md doc . Comment here Once it's done.

Looking forward to see your contributions to this project. Star this project if you find this is interesting.

Happy Coding!!
Happy Contributing!!